Formula/sshw.rb

```diff
@@ -0,0 +1,24 @@
+class Sshw < Formula
+  desc "🐝  ssh client wrapper for automatic login"
+  homepage "https://github.com/yinheli/sshw"
+  url "https://github.com/yinheli/sshw/archive/v1.0.4.tar.gz"
+  sha256 "7b48c8126f4fca50c186f864c5e9db1578d7670e2b9b0887922d5a2c56051a71"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["SSHW_GIT_TAG"] = "v1.0.4"
```

Comment:
I'd be temped to just build from a git checkout here, personally, rather than mandating someone manually updating the `GIT_HASH` env every single time.

Reply:
git checkout will need the hash updated too, and it disables caching.

Reply:
git checkout will need the hash updated too, and it disables caching.

Reply:
Doesn't the upstream `Makefile` calculate `SSHW_GIT_HASH` though, effectively removing our need to do it manually?

>  and it disables caching.

Don't really understand what you're saying here. We still cache git checkouts, AFAIK?

Reply:
Can't we replace it with `version` instead ?